app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __repr__(self):
+        return "{" + f"Name: {self.name}," \
+                     f" Health: {self.health}," \
+                     f" Hidden: {self.hidden}" + "}"
+
+    def __init__(self, name, health=100, hidden=False):
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+
+class Herbivore(Animal):
+
+    def hide(self):
+        if self.hidden:
+            self.hidden = False
```

Comment:
it may be done in a more simple way - reverse

Reply:
Agree